src/marqo/core/exceptions.py

```diff
@@ -12,6 +12,18 @@ class IndexNotFoundError(MarqoError):
     pass
 
 
+class IndexCreationAndDeletionConflictError(MarqoError):
```

Comment:
Maybe something more generic like `ConcurrencyConflictError`. Not the best name but you get the idea

Reply:
I make it `OperationConflictError`